Chirp/src/Chirp.Core/ICheepRepository.cs

```diff
@@ -6,5 +6,6 @@ public interface ICheepRepository
 {
     public Task<IEnumerable<CheepDTO>?> GetCheepsByPage(int page, int pageSize);
     public Task<IEnumerable<CheepDTO>> GetCheepsFromAuthorByPage(String author, int page, int pageSize);
+    public Task<IEnumerable<CheepDTO>> GetCheepsFromAuthorsByPage(IEnumerable<String> author, int page, int pageSize);
```

Comment:
For better clarification should 'author' be renamed to 'authors'? Also to differ more from the method on line 8

Reply:
Yes